lib/sentimental.rb

```diff
@@ -1,23 +1,23 @@
+require 'json'
+
 class Sentimental
   attr_accessor :threshold, :word_scores, :neutral_regexps, :ngrams
 
   def initialize(threshold: 0, word_scores: nil, neutral_regexps: [], ngrams: 1)
-    if ngrams >= 1
-      @ngrams = ngrams.to_i
-    else
-      @ngrams = 1
-    end
+    @ngrams = ngrams.to_i if ngrams >= 1
+    @ngrams = 1 unless ngrams >= 1
```

Comment:
I don't really like the first one since it initialize something outside of initialize (which looks like a terrible idea).

And, for me, there's a special place in hell for people who uses ternary if 😉